### ISSUE TITLE
Report specific CI environment in the header

### ISSUE THREAD

**Issue by laurentsigal** [waldoapp/waldo-cli#9]
Right now, we simply send the binary with:
```
User-Agent: Waldo CLI/iOS v1.2.3
```
If we're able to detect the exact environment (for instance fastlane) we should instead send using:
```
User-Agent: Waldo Bitrise v1.2.3
```

**Comment by eBardX** [comment]
@laurentsigal 

Is this _in addition to_ or _instead of_ uploading the CI name as part of the error message payload?

**Comment by laurentsigal** [comment]
in addition

**Comment by eBardX** [comment]
Implemented in Waldo CLI 1.6.1, available in store now!
